Microsoft.Toolkit.Uwp.Notifications/DesktopNotificationManager/DesktopNotificationManagerCompat.cs

```diff
@@ -2,14 +2,33 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+/*
```

Comment:
It's not just a single function, it's...

1. Some new private constants
1. Changed one private method call
1. New private IClassFactory interface
1. New private NotificationActivatorClassFactory class
1. New private DLL import

It's mostly grouped together in one section, but not sure it'd make sense having it down there since it is a bit spread out?

Maybe I could put the license at the RegisterActivator method and state that the license is for that method AND all dependent methods?

I agree about linking to the ThirdPartyNotice, makes sense!